mantidproject/mantid
# Only use drag-n-drop to load PeaksWorkspaces in the VSI.

Description of work.

It is easy to accidentally load a PeaksWorkspace before loading an MDWorkspace in the VSI, which will then not be displayed correctly. I removed this error-prone option and added a check that another workspace was loaded ahead of the PeaksWorkspace. Users should now drag the PeaksWorkspace into the VSI window, just like they already do with the SliceViewer.

**To test:**

Load a peaks workspace and verify that the dock option to load it in the VSI has been removed. Confirm that it can still be loaded by dragging it into the VSI window. Delete all sources in the VSI and confirm you can no longer drag-n-drop a peaks workspace into the VSI window.



This is a small change with no issue number.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
